docs/trace/README.md

```diff
@@ -1,19 +1,122 @@
 # OpenTelemetry .NET Traces
 
+<!-- markdownlint-disable MD033 -->
+<details>
+<summary>Table of Contents</summary>
+
+* [Best Practices](#best-practices)
+* [Package Version](#package-version)
+* [Tracing API](#tracing-api)
+* [TracerProvider Management](#tracerprovider-management)
+* [Correlation](#correlation)
```

Comment:
Should we add an entry for "Common issues that lead to missing traces" it uses same heading level.

Reply:
I intended to leave it out at this moment:
1. I think that section needs to be restructured.
2. It seems like a better fit for a troubleshooting doc instead of guidance doc.